armi/settings/fwSettings/globalSettings.py

```diff
@@ -209,64 +211,67 @@ def defineSettings() -> List[setting.Setting]:
         ),
         setting.Setting(
             CONF_CYCLE_LENGTH,
-            default=365.242199,
+            default=None,
             label="Cycle Length",
-            description="Duration of one single cycle in days. If availability factor is below "
+            description="Duration of one single cycle in days. If `availabilityFactor` is below "
             "1, the reactor will be at power less than this. If variable, use "
-            "cycleLengths setting.",
+            "`cycleLengths` setting.",
             oldNames=[
                 ("burnTime", None),
             ],
+            schema=(vol.Any(float, int, None)),
         ),
         setting.Setting(
             CONF_CYCLE_LENGTHS,
-            default=[],
+            default=None,
             label="Cycle durations",
             description="List of durations of each cycle in days. The at-power "
-            "duration will be affected by the availability factor. R is repeat. For "
+            "duration will be affected by `availabilityFactor`. R is repeat. For "
             "example [100, 150, '9R'] is 1 100 day cycle followed by 10 150 day "
-            "cycles. Empty list is constant duration set by 'cycleLength'.",
-            schema=vol.Schema([vol.Coerce(str)]),
+            "cycles. Empty list is constant duration set by `cycleLength`.",
+            schema=vol.Any([vol.Coerce(str)], None),
         ),
         setting.Setting(
             CONF_AVAILABILITY_FACTOR,
-            default=1.0,
+            default=None,
             label="Plant Availability Factor",
             description="Availability factor of the plant. This is the fraction of the "
-            "time that the plant is operating. If variable, use availabilityFactors "
+            "time that the plant is operating. If variable, use `availabilityFactors` "
             "setting.",
             oldNames=[
                 ("capacityFactor", None),
             ],
+            schema=(vol.Any(float, int, None)),
         ),
         setting.Setting(
             CONF_AVAILABILITY_FACTORS,
-            default=[],
+            default=None,
```

Comment:
This default has the wrong type. It _looks_ invalid.

Perhaps you meant to just remove the default value?

Reply:
Outdated. This was unwound when the settings defaults were reinstated.